webgui/package.json

```diff
@@ -15,7 +15,8 @@
     "svg-pan-zoom"     : ">=2.19.0",
     "marked"           : ">=0.3.6",
     "jsplumb"          : "2.2.1",
-    "jquery"           : ">=3.1.1"
+    "jquery"           : ">=3.1.1",
+    "js-base64"        : ">=2.5.2"
```

Comment:
So I should use ^2.5.0 for js-base64? Or ^2.0.0? I did not touch which jquery, the separator , is just needed to make the json file syntactically correct.

Reply:
This is not a problem in your pull request, but I think we should use `^` instead of `>=` everywhere not only for `js-base64` (in your case you can use `"^2.5.2"`). So these changes can be done in a separate pull request or here.

Reply:
Removing base64 encoding from username and password made this change obsolete.